doc/versioner.md

```diff
@@ -0,0 +1,91 @@
+# Versioning a project
+
+Executing `pkgmt version` will modify the `__version__` string in the version file, and the headers in `CHANGELOG.md`
+
+### Project Structure
+Currently, `pkgmt` supports versioning in two types of projects:
+
+* Projects which contain a `setup.py` file. If multiple directories found in src folder the first is selected for versioning.
+
+An example project structure is given below:
+
+```
+project_root
+|
+|__src
+   |___dir1
+       |___  __init__.py
+   |
+   |___dir2
+|
+|__CHANGELOG.md
+|
+|__pyproject.toml
+```
+
+`pyproject.toml` should contain an entry like:
+
+```
+[tool.pkgmt]
+github = "project/repository" 
+```
+* Projects which do not contain a `setup.py` file. In this case the `pyproject.toml` should contain an entry like:
+
+```
+[tool.pkgmt]
+github = "project/repository" 
+version_file = "/path/to/version_file.py"
```

Comment:
one last thing, let's modify this so the format is:

```toml
[tool.pkgmt]
github = "project/repository" 
version = {version_file: "/path/to/version_file.py"}
```

this way, we can add new keys to the dictionary to customize the behavior of the `pkgmt version` command (like passing the push or tag arguments)